src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -898,6 +960,29 @@ public String toCommandLineString(){
             }
         }
 
+        /**
+         * @param argumentDefinitionMap a map from argument names to {@link ArgumentDefinition}
+         * @return the set of arguments that this argument is mutually exclusive with (including itself in the set)
+         */
+        public Set<ArgumentDefinition> getMutuallyExclusiveArguments(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            final Set<ArgumentDefinition> mutexArguments = mutuallyExclusive.stream().map(argumentDefinitionMap::get).collect(Collectors.toSet());
+            mutexArguments.add(this);
+            return mutexArguments;
+        }
+
+        public String composeMissingArgumentMessage(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            if(mutuallyExclusive.isEmpty()){
+                return String.format("required argument --%s was not specified", getLongName());
```

Comment:
also, I noticed that "This argument must be specified at least once." is printed when using --help  and when the argument is a collection (The statement is useless because help also says `required`).